lib/helpers/model/applyStaticHooks.js

```diff
@@ -45,7 +45,9 @@ module.exports = function applyStaticHooks(model, hooks, statics) {
         // Special case: can't use `Kareem#wrap()` because it doesn't currently
         // support wrapped functions that return a promise.
```

Comment:
As a aside, is this still true?

Reply:
Correct, this statement is no longer true, it looks like https://github.com/mongoosejs/kareem/commit/c9d7dd1e249fc1f309f8d688d22fe725cb469921 added support for this.